source/usage-examples/command.txt

```diff
@@ -4,16 +4,52 @@ Run a Command
 
 .. default-domain:: mongodb
 
-You can run all raw database operations using the
-``MongoDatabase.runCommand()`` method. A raw database operation is a
-command you can execute directly on the MongoDB server CLI. These
-commands include administrative and diagnostic tasks, such as fetching
-server stats or initializing a replica set. Call the ``runCommand()``
-method with a Bson command object on an instance of a ``MongoDatabase``
-to run your raw database operation.
+You can run commands directly on the MongoDB server CLI using the
```

Comment:
```suggestion
You can run commands directly on your MongoDB server using the
```

Question (possible Issue): This sounds odd, as "CLI" implies a Command Line, and I don't think this is executed from a command line (I think the server just runs the command). I see this is also the phrasing in the Java Docs, however it does not seem to be the phrasing in the Node Docs or the [Mongosh docs](https://docs.mongodb.com/manual/reference/method/db.runCommand/). Can this be double checked?